calico_cni_k8s_test.go

```diff
@@ -82,6 +84,26 @@ var _ = Describe("CalicoCni", func() {
 				if err != nil {
 					panic(err)
 				}
+
+				// Now create a K8s pod passing in an IP pool
+				name2 := name + "-pool"
+				pod, err := clientset.Pods(K8S_TEST_NS).Create(&v1.Pod{
+					ObjectMeta: v1.ObjectMeta{
+						Name: name2,
+						Annotations: map[string]string{
+							"cni.calico/ipv4pools": "192.169.1.0/24",
```

Comment:
This annotation won't be used since you're calling CreateContainer below with `name` not `name2`